packages/orchestration/src/cosmos-api.ts

```diff
@@ -353,10 +353,10 @@ export interface IBCMsgTransferOptions {
 export type CosmosChainAccountMethods<CCI extends CosmosChainInfo> =
   IcaAccountMethods &
     (CCI extends {
-      stakingTokens: {};
+      stakingTokens: object;
     }
       ? StakingAccountActions & StakingAccountQueries
-      : {});
+      : object);
```

Comment:
Nice, [TIL](https://typescript-eslint.io/rules/no-empty-object-type/): `{} represents any non-nullish value, including literals like 0 and ""`

Reply:
yeah me too. quite surprising.